### DISCUSSION TITLE
Versatiles as background map

### DISCUSSION THREAD

**Discussion by 3D-Schorsch** [versatiles-org/versatiles-documentation#17] [Q&A] [answered]
How can I use versatiles as a background map in e.g. Umap? Is there a link in the scheme "http://{s}.domain.com/{z}/{x}/{y}.png"?

**Comment by MichaelKreil** [comment, marked as answer]
We serve vector tiles, so no PNGs.
But I have just written a few lines about how to use our public tile server:
https://github.com/versatiles-org/versatiles-documentation/blob/main/guides/use_tiles.versatiles.org.md

**Reply by 3D-Schorsch** [reply]
What I also noticed is that this link leads to nothing: https://tiles.versatiles.org/tiles/osm/{z}/{x}/{y} when I enter data there. Here is my example: https://tiles.versatiles.org/tiles/osm/#7/50.007/11.405

**Reply by MichaelKreil** [reply]
It is not a link, but an URL template (z=zoom, x=tile column, y=tile row)
So a correct URL would be: https://tiles.versatiles.org/tiles/osm/7/68/44
